src/main/scala/nl.knaw.dans.easy.deposit/package.scala

```diff
@@ -71,4 +74,19 @@ package object deposit {
         .getOrElse(onSuccess)
     }
   }
+
+  @throws[IOException]("when files can not be moved atomically from src to target")
+  def sameMounts(srcProvider: FileSystemProvider, src: File, target: File): Unit = {
```

Comment:
I'm not yet a big fan of putting this validation in the package object. Could you think of another (dedicated) place for these kinds of startup validations?